src/ui-test/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { BottomBarPanel, InputBox, WebDriver, Workbench } from "vscode-uitests-tooling";
+
+export const TEST_ARRAY_RUN = [
+    "Routes startup",
+    "Hello Camel from yaml"];
+
+export const CAMEL_RUN_DEBUG_ACTION_LABEL = 'Run Camel Application with JBang and Debug';
+export const CAMEL_RUN_ACTION_LABEL = 'Run Camel Application with JBang';
+export const CAMEL_ROUTE_YAML_WITH_SPACE = 'demo route.camel.yaml';
+
+/**
+ * Executes a command in the command prompt of the workbench.
+ * @param command The command to execute.
+ * @returns A Promise that resolves when the command is executed.
+ */
+export async function useCommandInPalette(command: string): Promise<void> {
+    const workbench = new Workbench();
+    await workbench.openCommandPrompt();
+    const input = await InputBox.create();
+    await input.setText(`>${command}`);
+    await input.confirm();
+}
+
+/**
+ * Checks if the terminal view has the specified texts in the given textArray.
+ * @param driver The WebDriver instance to use.
+ * @param textArray An array of strings representing the texts to search for in the terminal view.
+ * @param interval (Optional) The interval in milliseconds to wait between checks. Default is 1000ms.
+ * @returns A Promise that resolves to a boolean indicating whether the terminal view has the texts or not.
+ */
+export async function waitUntilTerminalHasText(driver: WebDriver, textArray: string[], interval = 1000): Promise<void> {
```

Comment:
maybe decrease time of interval to `500ms`?.. you know it's still terminal log and its quite fast, we can save some testing time here if I will consider this will be used across multiple test classes